.github/workflows/ci.yaml

```diff
@@ -15,6 +15,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/setup-go@v2
+      - run: go get golang.org/x/tools/cmd/stringer
```

Comment:
@g4s8, `go get golang.org/x/tools/cmd/stringer` do not download _stringer_ executable. Installing of `golang-golang-x-tools` helps to run `make` successfully on ubuntu20.04. We can add it to the makefile somehow...

Reply:
@ivanbukhtiyarov actually it does, the problem was in actions syntax: I replaced `setup-go` action config with `go get` command, this is why go was not installed correctly. Now it's working fine.